crates/rattler_virtual_packages/src/lib.rs

```diff
@@ -130,11 +176,19 @@ impl From<VirtualPackage> for GenericVirtualPackage {
 impl VirtualPackage {
     /// Returns virtual packages detected for the current system or an error if the versions could
     /// not be properly detected.
-    pub fn current() -> Result<&'static [Self], DetectVirtualPackageError> {
-        static DETECTED_VIRTUAL_PACKAGES: OnceCell<Vec<VirtualPackage>> = OnceCell::new();
-        DETECTED_VIRTUAL_PACKAGES
-            .get_or_try_init(try_detect_virtual_packages)
-            .map(Vec::as_slice)
+    #[deprecated(
+        since = "1.0.4",
+        note = "Use `Self::detect(&Default::default)` instead"
```

Comment:
mm I guess this should be none right?

Reply:
I'd argue no, because the expectation is to be conda compatible :shrug:

Reply:
I disagree, rattler is also used in applications where this outside environment override behavior is unwanted. For an application this behavior makes sense but for a library I think reading from environment variables should be opt in.

Reply:
Furthermore, this suggestion changes the default current behavior of the function. I think the depreciation should tell people how to migrate to something which exhibits the same behavior.